pyrsia_cli/src/cli/parser.rs

```diff
@@ -82,12 +82,14 @@ pub fn cli_parser() -> ArgMatches {
                         .arg_required_else_help(true)
                         .args(&[
                             arg!(--image <IMAGE> "The docker image (e.g. alpine:3.15.3 or alpine@sha256:1e014f84205d569a5cc3be4e108ca614055f7e21d11928946113ab3f36054801"),
+                            arg!(--format <FORMAT> "The output format (json, csv)"),
```

Comment:
I would make "json" the default value and explicitly define the possible values with the argument:
```suggestion
                            arg!(--format <FORMAT> "The output format")
                                .value_parser(["json", "csv"])
                                .default_value("json"),
```

Reply:
great